Makefile

```diff
@@ -38,153 +42,172 @@ help:  ## Display this help
 	@echo "  GO_TEST_FLAGS    -- flags to pass to --go-test-flags ($(GO_TEST_FLAGS))"
 	@echo "  DEBUG            -- debug flag, if any ($(DEBUG))"
 
-.PHONY: $(KUSTOMIZE)
-$(KUSTOMIZE):
-	cd $(TOOLS_DIR); ./install_kustomize.sh
+# Image URL to use all building/pushing image targets
+IMG ?= controller:latest
```

Comment:
This should probably be baremetal-operator, to stay close to quay.io/metal3-io/baremetal-operator, no ?

Reply:
Done